pkgs/development/ada-modules/gnatcoll/bindings.nix

```diff
@@ -42,6 +43,24 @@ stdenv.mkDerivation rec {
     sha256 = "00aakpmr67r72l1h3jpkaw83p1a2mjjvfk635yy5c1nss3ji1qjm";
   };
 
+  patches = [
+    (fetchpatch {
+      # Add minimal support for Python 3.11
+      url = "https://github.com/AdaCore/gnatcoll-bindings/commit/cd650de5.patch";
+      hash = "sha256-4zHTbUnwKdEMpT/KERsZOEN0/QbVqQHboPKKea9IPiA=";
+    })
+    (fetchpatch {
+      # Add a build node in gitlab CI (needed for the patch right after)
+      url = "https://github.com/AdaCore/gnatcoll-bindings/commit/ad58af47.patch";
+      hash = "sha256-ljqBT+B7SH5BUiM8pi4NxW9OP3CXug3ZwTP/OWgcb+s=";
+    })
```

Comment:
This patch can be skipped by adding `excludes = [ ".gitlab-ci.yml" ];` to the next one (and recalculating the hash).